debian/conf/jellyfin.init

```diff
@@ -19,13 +19,14 @@ fi
 
 . /lib/lsb/init-functions
 
+JELLYFIN_HOME="/var/lib/jellyfin/"
```

Comment:
This definitely should not be defined statically in here. If this is to be set, it must be set in `/etc/default/jellyfin` so that administrators (e.g. me) can override it.